providers/amazon/src/airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -59,6 +59,7 @@
 from airflow.providers.amazon.aws.utils.identifiers import generate_uuid
 from airflow.providers.amazon.aws.utils.suppress import return_on_error
 from airflow.providers_manager import ProvidersManager
+from airflow.sdk.exceptions import AirflowRuntimeError
```

Comment:
This needs to be under version check as needs to be compatible with Airflow 2.10.0 and sdk path did not exist in 2.10

Reply:
Had to rebase, fixed this in [3779cee](https://github.com/apache/airflow/pull/50515/commits/3779cee61c627831d9672f239fa33eb6d641844d).  Thanks!